README.md

```diff
@@ -1 +1,136 @@
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Book Store: <a name="about-project"></a>
+
+> Book Store is a web application that allows users to add and remove books from a list. The application is built with React and Redux. The main goal is to put into practice the concepts of React and Redux and also to use the best practices in React programming.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- BUILT WITH -->
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Programming languages: 
+    JSX, CSS3, React
+- Tools: 
+    Git, GitHub, Linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- Dynamic webpage
+- Built with react components
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Link to deployed project.
+
+- [Live Demo Link](Coming soon)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Git and Code editor.
+
+### Setup
+
+Clone this repository to your desired folder:
+``git clone `https://github.com/yodit93/book-store.git`
+Install necessery dependencies using the following syntax.
+- npm install
+Finally run the app using the following syntax.
+- npm start
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Future Features <a name="key-features"></a>
+- Add functionalities to add and remove books.
+- Implement some UX improvements: transitions and animations.
+- Add more tests.
+- Add more features to the app.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Yodit Abebe
+
+- GitHub: [yodit93](https://github.com/yodit93)
+- Twitter: [@yodtwit](https://twitter.com/yodtwit)
+- LinkedIn: [Yodit Abebe](linkedin.com/in/yodit-abebe-a97a68140)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/yodit93/book-store/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project please give a ⭐️ to this repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> I would like to thank [Microverseinc](https://github.com/microverseinc).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Good job Implementing the project requirements, everything is working well, however I cannot yet approve the project, as currently you have a broken MIT license link present in your Readme file and in order to have professional documentation of your work you should fix this and make sure to not have any broken links in your Readme, so please fix it by adding a License file : 🙏

Reply:
Hello @PinkMoon25 
Thank you very much
I included the license file in the development branch by mistake. That is why the link is broken on the current branch.
I have included in this branch now.